source/frontend/src/components/propertySelector/search/PropertySelectorPidSearchContainer.tsx

```diff
@@ -22,9 +25,16 @@ export const PropertySelectorPidSearchContainer: React.FunctionComponent<
   const searchFunc = useCallback(
     async (layerSearch: ILayerSearchCriteria) => {
       if (layerSearch?.pid) {
-        const result = await getPropertyByPidWrapper.execute(layerSearch?.pid);
-        if (result) {
-          setSelectProperty(result);
+        try {
+          const result = await getPropertyByPidWrapper.execute(layerSearch?.pid);
```

Comment:
this is the underlying story that brought up 7875 - when a property is searched for but does not exist - the error reporting is not clear.